packages/inter-protocol/src/proposals/demoIssuers.js

```diff
@@ -261,7 +262,8 @@ export const connectFaucet = async ({
         // TODO: what happens if faucetRunSupply doesn't have enough
         // remaining?
         let fragment;
-        [fragment, faucetRunSupply] = await E(runIssuer).split(
+        [fragment, faucetRunSupply] = await split(
+          E(runIssuer).makeEmptyPurse(),
```

Comment:
This should just use a purse and withdraw. But that's orthogonal to this PR.

Reply:
Agreed. And it is only a demo. Resolving.